src/Installer/dotnetup/Commands/SetInstallRoot/SetInstallRootCommandParser.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+
+namespace Microsoft.DotNet.Tools.Bootstrapper.Commands.SetInstallRoot;
+
+internal static class SetInstallRootCommandParser
+{
+    public static readonly Argument<string> InstallTypeArgument = new("installtype")
```

Comment:
Call the AcceptOnlyFromAmong method here to list the valid argument values.  They should be defined as constants in this file.  Add "admin" in addition to "user" as a valid value.